distributed/tests/test_client.py

```diff
@@ -1007,14 +1007,20 @@ async def test_map_quotes(c, s, a, b):
     assert all(result)
 
 
+def randomint(low, high):
+    # Serializing random.randint itself generates unique objects which the
+    # scheduler will reject
```

Comment:
This sounds like a substantial regression to me? random.randint is not exactly an obscure function.

Reply:
frankly, I don't know if this explanation is entirely true. the point is that this test case is very artificial

Reply:
Is it? A user's client may spawn multiple worker-clients, each generating a graph that is identical in the beginning but then diverges afterwards depending on the different tasks.

Sounds like a fairly realistic use case to me?

Reply:
Fixed in https://github.com/dask/dask/pull/10865